panelapp/panels/models/genepanelsnapshot.py

```diff
@@ -741,209 +665,13 @@ def increment_version(
                 # check if there are any parent panels
                 if super_panels:
                     for panel in GenePanelSnapshot.objects.filter(pk__in=super_panels):
-                        panel.increment_version(major=major, remove_child=old_pk)
+                        increment_panel_async.delay(user.pk, panel.pk, major=major)
 
             if getattr(self.panel, "active_panel"):
                 del self.panel.active_panel
 
             return self.panel.active_panel
```

Comment:
actually, this is no longer required since `active_panel` is always the last one, i.e. we only have a single snapshot panel